packages/firebase_dynamic_links/README.md

```diff
@@ -106,8 +106,21 @@ Remember not to include `https://` or any slashes or paths in your prefix
   <string>https://example.com/links/share</string>
 </array>
 ```
+5. If you want to open android app directly without redirecting to Chrome or Any other Browser you can put an intent-filter in your ```AndroidManifest.xml```. It will open system dialogue to open link with your app or other browsers. Users can directly open link in your app.
```

Comment:
```suggestion
5. If you want to open android app directly without redirecting to Chrome or Any other Browser you can put an `intent-filter` in your `AndroidManifest.xml` file. It will open the system dialogue to open link with your app or other browsers. Users can then directly choose to open the link in your app.
```